dspace-server-webapp/src/test/java/org/dspace/app/rest/CollectionGroupRestControllerIT.java

```diff
@@ -0,0 +1,2094 @@
+
+        getClient(token).perform(post("/api/core/collections/" + collection.getID() + "/adminGroup")
+                                     .content(mapper.writeValueAsBytes(groupRest))
+                                     .contentType(contentType))
+                        .andExpect(status().isUnprocessableEntity());
+
+    }
+
+    @Test
+    public void deleteCollectionAdminGroupTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        Group adminGroup = collectionService.createAdministrators(context, collection);
+        context.restoreAuthSystemState();
+
+        String token = getAuthToken(admin.getEmail(), password);
+        getClient(token).perform(delete("/api/core/collections/" + collection.getID() + "/adminGroup"))
+                        .andExpect(status().isNoContent());
+
+        getClient(token).perform(get("/api/core/collections/" + collection.getID() + "/adminGroup"))
+                        .andExpect(status().isNoContent());
+    }
+
+
+    // This is currently not supported in DSpace API
+    @Ignore
+    @Test
+    public void deleteCollectionAdminGroupTestParentCommunityAdmin() throws Exception {
+        Group adminGroup = collectionService.createAdministrators(context, collection);
+        authorizeService.addPolicy(context, parentCommunity, Constants.ADMIN, eperson);
+
+        String token = getAuthToken(eperson.getEmail(), password);
+        getClient(token).perform(delete("/api/core/collections/" + collection.getID() + "/adminGroup"))
+                        .andExpect(status().isNoContent());
+
+        getClient(token).perform(get("/api/core/collections/" + collection.getID() + "/adminGroup"))
+                        .andExpect(status().isNoContent());
+    }
+
+    // This is currently not supported in DSpace API
+    @Ignore
+    @Test
+    public void deleteCollectionAdminGroupTestCollectionAdmin() throws Exception {
+        Group adminGroup = collectionService.createAdministrators(context, collection);
+        authorizeService.addPolicy(context, collection, Constants.ADMIN, eperson);
+
+        String token = getAuthToken(eperson.getEmail(), password);
+        getClient(token).perform(delete("/api/core/collections/" + collection.getID() + "/adminGroup"))
+                        .andExpect(status().isNoContent());
+
+        getClient(token).perform(get("/api/core/collections/" + collection.getID() + "/adminGroup"))
+                        .andExpect(status().isNoContent());
+    }
+
+    @Test
+    public void deleteCollectionAdminGroupUnAuthorizedTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+        Group adminGroup = collectionService.createAdministrators(context, collection);
+        context.restoreAuthSystemState();
+
+        getClient().perform(delete("/api/core/collections/" + collection.getID() + "/adminGroup"))
+                   .andExpect(status().isUnauthorized());
```

Comment:
double check that the group is still here

Reply:
An extra get was added.